auth/src/main/java/org/aerogear/auth/AuthService.java

```diff
@@ -21,14 +24,15 @@
 
     /**
      * Instantiates a new AuthService object
-     * @param config Authentication Service configuration
      */
-    private AuthService(final AuthServiceConfig config) {
+    public AuthService() {}
+
+    public void bootstrap(final MobileCore core, final ServiceConfiguration serviceConfig) {
```

Comment:
When is the bootstrap called?
If we don't initialise the object inside the constructor, we should than throw an IllegalStateException if other methods are called before the bootstrap one.

Reply:
@secondsun Am I correct in saying the core will initialize and then call bootstrap on a ServiceModule?

Reply:
@aidenkeating Yes.  https://github.com/aerogear/proposals/blob/master/android-sdk/README.md#starting-up-mobile-core